runtime/compiler/optimizer/CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ j9jit_files(
 	optimizer/IdiomTransformations.cpp
 	optimizer/InlinerTempForJ9.cpp
 	optimizer/InterProceduralAnalyzer.cpp
+        optimizer/J9CFGSimplifier.cpp
```

Comment:
Please indent with a tab like the rest of this file.

Reply:
done